src/components/Footer/index.jsx

```diff
@@ -1,7 +1,41 @@
 import "./style.css";
+import discord from "../../assets/discord.svg";
+import facebook from "../../assets/fb_icon.svg";
+import instagram from "../../assets/insta_icon.svg";
+import mail from "../../assets/mail_icon.svg";
 
 const Footer = () => {
-  return <div></div>;
+  return (
+    <div id="footer">
+      <div>
+        <h1 id="h1">
+          Contact Us
+        </h1>
+      </div>
+        <div id="icon-box">
+            <div id="discord" className="icon">
+              <a href="" target="_blank" rel="noreferrer">
```

Comment:
can you keep some temporary value in the href (like #footer) and make sure you don't get any warnings in here when you run it? the website preview is currently failing if we have any warnings so i can't preview it :(
```
12:27:03 PM: src/components/Footer/index.jsx
12:27:03 PM:   Line 17:15:  The href attribute requires a valid value to be accessible. 
Provide a valid, navigable address as the href value. If you cannot provide a valid href, 
but still need the element to resemble a link, use a button and change it with appropriate
styles. Learn more: 
https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/HEAD/docs/rules/anchor-is-valid.md  jsx-a11y/anchor-is-valid
```